contrib/lite-setup.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python3
+#
+# Copyright (C) 2019 FreeIPA Contributors see COPYING for license
+#
+"""In-tree development server setup
+"""
+import argparse
+import os
+import socket
+from urllib.request import urlopen
+
+DEFAULT_CONF = """\
+[global]
+host = {args.hostname}
+server = {args.servername}
+basedn = {args.basedn}
+realm = {args.realm}
+domain = {args.domain}
+# xmlrpc_uri = https://{args.servername}/ipa/xml
+xmlrpc_uri = http://localhost:8888/ipa/xml
+ldap_uri = ldap://{args.servername}
+mode = production
+debug = False
+enable_ra = True
+ra_plugin = dogtag
+dogtag_version = 10
+"""
+
+KRB5_CONF = """\
+[libdefaults]
+ default_realm = {args.realm}
+ dns_lookup_realm = false
+ dns_lookup_kdc = false
+ rdns = false
+ ticket_lifetime = 24h
+ forwardable = true
+ udp_preference_limit = 0
+ default_ccache_name = FILE:{args.ccache}
+
+[realms]
+ {args.realm} = {{
+  kdc = {args.servername}:88
+  master_kdc = {args.servername}:88
+  admin_server = {args.servername}:749
+  # kdc = https://{args.servername}/KdcProxy
+  # master_kdc = https://{args.servername}/KdcProxy
+  # admin_server = https://{args.servername}/KdcProxy
+  default_domain = ipa.example
+  pkinit_anchors = FILE:{args.ca_crt}
+  pkinit_pool = FILE:{args.ca_crt}
+  http_anchors = FILE:{args.ca_crt}
+}}
+
+[domain_realm]
+ .ipa.example = {args.realm}
+ ipa.example = {args.realm}
+ {args.servername} = {args.realm}
+"""
+
+ACTIVATE = """\
+deactivate_ipaenv () {{
```

Comment:
Would it be worthwhile to use deactivate like virtual-env? Would someone want to run this within a venv?

Reply:
I selected another name to avoid conflicts with virtual environments.